pkgs/development/python-modules/scholarly/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, unittestCheckHook
+
+, arrow
+, beautifulsoup4
+, bibtexparser
+, deprecated
+, fake-useragent
+, free-proxy
+, httpx
+, python-dotenv
+, selenium
+, sphinx-rtd-theme
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "scholarly";
+  version = "1.7.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-LJg91E2dk5im8mBRAq5uVYYCO0HruuwUYZF+5I6xU/A=";
+  };
+
+  propagatedBuildInputs = [
+    arrow
+    beautifulsoup4
+    bibtexparser
+    deprecated
+    fake-useragent
+    free-proxy
+    httpx
+    python-dotenv
+    selenium
+    sphinx-rtd-theme
+    typing-extensions
+  ];
+
+  doCheck = false;
```

Comment:
Can we really not enable any tests? Python packages having tests enabled are a lot easier to keep properly working according to the python maintainers. Otherwise people may be flying blind a bit more.

Reply:
Enabled tests as suggested.